Source/Controller/HomeViewController.swift

```diff
@@ -265,9 +265,6 @@ class HomeViewController: ContentViewController, HelpDrawerHost {
                 }
             }
             AppController.shared.operationQueue.addOperation(operation)
-        } else {
```

Comment:
This could result in several calls to `load` executing simultaneously, especially when the app is first launching. I couldn't find any cases where this block was strictly needed since we call `load` in `viewDidLoad`. But if you know of one @martindsq let me know.

Reply:
This was added a long time ago when we wanted to automatically refresh the screen for new users that had an empty feed (because they didn't follow anyone yet and we didn't have welcome posts as we have now).
As you said, this is not needed anymore.